subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/internal/DefaultSwiftXCTestSuite.java

```diff
@@ -29,16 +31,22 @@
 import javax.inject.Inject;
 
 public class DefaultSwiftXCTestSuite extends DefaultSwiftComponent implements SwiftXCTestSuite {
+    private final Property<SwiftBinary> testedComponent;
     private final DefaultSwiftBundle bundle;
     private final DirectoryProperty resourceDirectory;
 
     @Inject
     public DefaultSwiftXCTestSuite(String name, ObjectFactory objectFactory, FileOperations fileOperations, ConfigurationContainer configurations, ProjectLayout projectLayout) {
         super(name, fileOperations, objectFactory, configurations);
 
+        testedComponent = objectFactory.property(SwiftBinary.class);
         resourceDirectory = projectLayout.directoryProperty();
         resourceDirectory.set(projectLayout.getProjectDirectory().dir("src/" + name + "/resources"));
-        bundle = objectFactory.newInstance(DefaultSwiftBundle.class, name + "Bundle", projectLayout, objectFactory, getModule(), true, getSwiftSource(), configurations, getImplementationDependencies(), getResourceDir());
+        bundle = objectFactory.newInstance(DefaultSwiftBundle.class, name + "Bundle", projectLayout, objectFactory, getModule(), true, getSwiftSource(), configurations, getImplementationDependencies(), getResourceDir(), getTestedComponent());
+    }
+
+    public Property<SwiftBinary> getTestedComponent() {
+        return testedComponent;
```

Comment:
Please wait for my PR to go in, it adds the tested component capability and would cause me to rebase yet another time.

Reply:
@lacasseio, you got your pull request in already, right?